AsyncDisplayKit/ASCellNode.mm

```diff
@@ -236,6 +236,12 @@ - (NSIndexPath *)indexPath
   return nil;
 }
 
+- (UIViewController *)viewController
+{
+  ASDisplayNodeAssertMainThread();
+  return _viewController;
```

Comment:
Would it be reasonable to force-load the view here, so that the cell node will run the view controller block if it hasn't already? It would be sufficient to add something like:

```objectivec
- (UIViewController *)viewController
{
  ASDisplayNodeAssertMainThread();
  // Force the view to load so that we will create the
  // view controller if we haven't already.
  if (self.isNodeLoaded == NO) {
    [self view];
  }
  return _viewController;
}
```